lib/Doctrine/Migrations/Generator/DiffGenerator.php

```diff
@@ -98,7 +94,7 @@ static function ($assetName) use ($filterExpression) {
         );
 
         if ($up === '' && $down === '') {
-            throw NoChangesDetected::new();
```

Comment:
We shouldn't change this, a try/catch block can be added to the command. That isolates the change and doesn't introduce a new type in the signature of the method (null).

Reply:
I restored this